`yarn build` fails to build graphql-serve

<!--
Thank you for reporting a possible bug in Graphback

Please fill in as much of the template below as you can.

Module: if known, please specify the affected module name. (examples: graphback-runtime-knex, graphback-runtime-mongodb, graphql-migrations etc)
Version: version of the affected module


If possible, please provide code that demonstrates the problem, keeping it as
simple and free of external dependencies as you can.
-->

Running `yarn build` from the root, sometimes graphql-serve does not build successfully.

* **Module**: graphql-serve
* **Version**: 0.16.x
* **Node.js / npm versions:** NA

```sh
$ tsc
lerna ERR! yarn run build exited 2 in 'graphql-serve'
lerna ERR! yarn run build stdout:
$ tsc
src/commands/printSchema.ts(2,10): error TS2614: Module '"../../../../node_modules/chalk/types"' has no exported member 'cyan'. Did you mean to use 'import cyan from "../../../../node_modules/chalk/types"' instead?
```

<!-- Please provide more details and steps to reproduce the issue below this comment. -->

--- a/packages/graphql-serve/src/commands/printSchema.ts
+++ b/packages/graphql-serve/src/commands/printSchema.ts
@@ -1,5 +1,5 @@
 import yargs from 'yargs';
-import { cyan } from 'chalk';
+import chalk from 'chalk';
 import { printSchemaHandler } from '../components';
 
 type Params = { model?: string };
@@ -22,5 +22,5 @@ export function handler(args: Params): void {
   const schemaSDL = printSchemaHandler(args);
 
   console.log("Generated schema:\n");
-  console.log(cyan(schemaSDL));
+  console.log(chalk.cyan(schemaSDL));
 }

